code/controllers/configuration.dm

```diff
@@ -120,8 +120,8 @@
 
 	//Used for modifying movement speed for mobs.
 	//Unversal modifiers
-	var/run_speed = 0
-	var/walk_speed = 0
+	var/run_speed = 10
```

Comment:
I'm fairly sure this will not work as you intend because they are in the configurations file